bolt/test/perf2bolt/X86/perf2bolt-spe.test

```diff
@@ -0,0 +1,9 @@
+## Check that Arm SPE mode is unavailable on X86.
+
+REQUIRES: system-linux,x86_64-linux
+
```

Comment:
This test now is failing. Probably this line was accidentally removed?

https://github.com/llvm/llvm-project/blob/089cfd0cd427f10013feef835fe181aa0c0459f3/bolt/test/perf2bolt/AArch64/perf2bolt-spe.test#L5